katz/src/test/kotlin/katz/data/IorTest.kt

```diff
@@ -132,5 +133,26 @@ class IorTest : UnitSpec() {
 
         }
 
+        "Ior.monad.flatMap should be consistent with Ior#flatMap" {
+            forAll { a: Int ->
+                val implicit: Semigroup<Number> = object: Semigroup<Number> {
+                    override fun combine(a: Number, b: Number): Number = a
+                }
```

Comment:
I agree with the confusion of implicit. Please also rename the other implicit named things on IorTest like the ones i added (on master now).